src/entities/camera.rs

```diff
@@ -1,25 +1,67 @@
 use amethyst::{
-    core::{Parent, Transform},
-    ecs::{prelude::World, Entity},
+    core::Transform,
+    core::math::Matrix4,
+    ecs::prelude::World,
     prelude::Builder,
     renderer::camera::Camera,
     window::ScreenDimensions,
 };
 
-pub fn load_camera(world: &mut World, camera_subject: Entity) {
+pub fn load_camera(world: &mut World) {
     let (width, height) = {
         let dim = world.read_resource::<ScreenDimensions>();
         (dim.width(), dim.height())
     };
     let mut transform = Transform::default();
-    transform.set_translation_xyz(0.0, 0.0, 1.0);
+    let x = 384.0;
+    let y = 176.0;
+    transform.set_translation_xyz(x, y, 1.);
+
+    let mut camera = init_parallax_camera(width, height);
+    set_paralax_offset(&mut camera, x, y);
 
     world
         .create_entity()
-        .with(Camera::standard_2d(width, height))
-        .with(Parent {
-            entity: camera_subject,
-        })
+        .with(camera)
         .with(transform)
         .build();
 }
+
+fn init_parallax_camera(width: f32, height: f32) -> Camera{
+    let mut camera = Camera::standard_2d(width, height);
+    // Set orthogonal projection matrix
+    let mut matrix = Matrix4::identity();
+    let z_near = 0.01;
+    let z_far = 2000.0;
+
+    matrix[(0, 0)] = 2.0/width;
+    matrix[(1, 1)] = -2.0/height;
+    matrix[(2, 2)] = -1.0 / (z_far - z_near);
+    matrix[(2, 3)] = -z_near / (z_far - z_near);
+
+    *camera.as_matrix_mut() = matrix;
+    set_paralax_offset(&mut camera, 0.0, 0.0);
+    camera
+}
+
+/// Add parallax translation to camera.
+///
+/// Camera assumed to be setup with `init_parallax_camera`
+/// (Currently equal to `Camera::standard_2d`)
+///
+/// If parallax translation being kept equal to camera translation
+/// it would cause further objects to move away slower (in screen coordinates)
+/// with infinitely distant objects being stationary (infinity=z_far=2000.0)
+pub fn set_paralax_offset(camera: &mut Camera, x: f32, y: f32){
```

Comment:
Nice work! This gets rid of some unnecessary stuff that I was doing. And thanks for adding the comments.